File: Server/Server/Models/Requests/RegisterRequest.cs
namespace Server.Models
{
    public class RegisterRequest
    {
        public string Email { get; set; }
        public string Password { get; set; }
        public string Name { get; set; }
        public int RoleId { get; set; }

        public RegisterRequest() 
        {
            Email = string.Empty;
            Password = string.Empty;
            Name = string.Empty;
            RoleId = 0;
        }
    }
}

File: Client/app/src/main/java/com/example/client/networking/requests/RegisterRequest.java
package com.example.client.networking.requests;

public class RegisterRequest {
    private String email;
    private String password;
    private String name;
    private int roleId;

    public RegisterRequest(String email, String password, String name, int roleId) {
        this.email = email;
        this.password = password;
        this.name = name;
        this.roleId = roleId;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getRoleId() {
        return roleId;
    }

    public void setRoleId(int roleId) {
        this.roleId = roleId;
    }
}
